files/zh-cn/web/javascript/reference/global_objects/reflect/set/index.md

```diff
@@ -10,7 +10,7 @@ slug: Web/JavaScript/Reference/Global_Objects/Reflect/set
 ## 语法
 
 ```plain
-Reflect.set(target, propertyKey, value[, receiver])
+Reflect.set(target, propertyKey, value, receiver)
 ```
```

Comment:
````suggestion
```js-nolint
Reflect.set(target, propertyKey, value)
Reflect.set(target, propertyKey, value, receiver)
```
````

Reply:
Thanks for explaining it to me